tests/unit/test_routes/test_searching.py

```diff
@@ -0,0 +1,274 @@
+from os import environ
+from app.utils import get_error_code_from_status, msg_map, random_string
+from .helpers import get_api_key
+
+
+def test_search(
+        module_client, module_db, fake_auth_from_oc, fake_algolia_save,
+        fake_algolia_search):
+    client = module_client
+
+    first_term = random_string()
+    apikey = get_api_key(client)
+
+    # Create resource and find it in the search results.
```

Comment:
This is kind of a funky set of tests because it's not actually using Algolia... Like, having an integration test would make more sense than a unit test for this. I think we may want to have a chat about what we're really testing here and take a second look at whether this accomplishes that in the best way. I'm honestly not sure, but I could see this being improved

Reply:
Maybe we could start a test automation project to run integration tests on the different parts of OC...